### DISCUSSION TITLE
TF coil: Resistive magnet Young's modulus

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#3298] [General]
In GitLab by @skahn on May 12, 2020, 16:27

## Description

The presence of inter-turn insulation is not present in the resistive stress calculation. The Young's modulus must be modified to take this into account. As the fraction taken by the insulation varies as a function of the radius, the corresponding Young's modulus will be radially dependent

## Proposed actions

* [x] Change the vertical young modulus to take the cooling and the insulations into account.
* [ ] Modify the toroidal smeared young modulus expression to take insulation layer as a function of R and see if an analytical solution still holds.
* [ ] Find a way to take the cooling channels effect on the toroidal/radial Young modulus.

Best regards     
Seb
Ref : @stuartmuldrew, @mkovari and @jmorris-uk

**Comment by jonmaddock** [comment]
In GitLab by @mkovari on May 12, 2020, 16:33

A diagram showing the cross-section of the resistive TF coil would be desirable.

**Comment by jonmaddock** [comment]
In GitLab by @skahn on May 23, 2020, 08:36

![image](https://git.ccfe.ac.uk/process/process/uploads/510ee8cfa81dee72ea1f17188b782071/image.png)

**Comment by jonmaddock** [comment]
In GitLab by @skahn on Feb 5, 2021, 13:53

marked the task **Change the vertical young modulus to take the cooling and the insulations.** as completed

**Comment by grmtrkngtn** [comment]
@mkovari Do you have any thoughts on whether this change is worth implementing in the stress calculation?

![image](https://github.com/user-attachments/assets/4947db7d-e4b4-48e7-b568-941446053c36)

We would need to develop a model for the positions of the cooling channels, to get a radially dependent calculation of Young's odulus - if I understand this correctly.

**Comment by mkovari** [comment]
> Modify the toroidal smeared young modulus expression to take insulation layer as a function of R and see if an analytical solution still holds.

> We would need to develop a model for the positions of the cooling channels, to get a radially dependent calculation of Young's modulus

Deriving a radially dependent Young's modulus may be easy, but obtaining an analytical solution looks very difficult to me unless you already understand Seb's mathematics, and maybe even then.

The insulation has two effects: (a) it changes the distribution of stress in the copper, and (b) it might itself fail at sufficiently high stress.  In the supercon TF calculation we include the first effect but not the second, since we don't calculate the stress in the insulator.

I suspect the presence of the insulation won't make much difference to the peak stress in the copper.

What it will do is act as a compliant layer, taking up any gap between the copper and the bucking cylinder, which is a good thing.

I am a bit worried the cooling channels could collapse.  It is straightforward to calculate the stress concentration around the hole due to the tangential and radial stress in the copper, working in two dimensions and ignoring the effect of the hole on the overall stress distribution.  One could then add the vertical stress as well to derive the maximum shear stress for the Tresca failure criterion.

Final question - is anyone interested in the resistive TF coil??
@ym1906

**Comment by grmtrkngtn** [comment]
Thank you for your comments Michael. After asking around, I think this is a large issue which should be converted to a discussion as it will require some careful planning and thought before initiating work on this task.
